assets/i18n/en.json

```diff
@@ -42,6 +42,7 @@
     "I18N_ACTION_BROWSE_LESSONS": "Browse our Lessons",
     "I18N_ACTION_CREATE_EXPLORATION": "Create an Exploration",
     "I18N_ACTION_CREATE_LESSON": "Create your own Lesson",
+    "I18N_CARD_HEIGHT_LIMIT_WARNING_MESSAGE": "This card is quite long, and students might lose interest. Consider shortening it, or breaking it into two cards.",
```

Comment:
This also needed to be added in qqq.json file.

Reply:
Done.